### DISCUSSION TITLE
Why can't I upload my transcript to github's student developer pack.

### DISCUSSION THREAD

**Discussion by akiy-33** [community/community#17762] [GitHub Education] [answered]
There used to be an upload button, now there is no upload button anymore. How am I supposed to upload it without a camera?
![Screenshot 2022-04-29 135037](https://user-images.githubusercontent.com/62551880/165930941-176667fa-d719-49c2-9987-9814592b3311.png)

**Comment by L3rpio** [comment]
It only accepts image file types, so you probably won't be able to upload your transcript, which is probably not an image file type. Try just taking a picture of your transcript and uploading that instead.

**Comment by elisemoe** [comment]
Hi @akiy-33  👋  Would you be able to apply from a phone camera?

Sometimes, for a number of reasons, our system needs additional levels of verification and asks that you take an image rather than upload one. Other suggestions might be to try applying while you are on campus, if able to.

**Reply by raghukovvuru** [reply]
Why don't you use something like SheerID like any sane company would do?

**Reply by marksubb** [reply]
Are there any plans to fix this issue?

FYI @elisemoe, attempting to do this from a phone does not work either (at least on my phone). There is still only the option to "Take a picture", which hangs indefinitely when I select it.

Additionally, as a remote student, the webpage is not allowing me to proceed because I am in a geographically different region than my school. Even if I were to take a ton of pictures and _finally_ get one that the system approves, I'm still in a different region so I'm out of luck. This is even with "Campus: online" being shown on my documents that I've attempted to upload.

+1 on the suggestion to use something like SheerID or Id Me. This webcam upload system does not seem very robust and I would love to hear from someone that actually managed to get approved through it.

**Comment by aidenh37** [comment]
Having the same issue. Require GitHub for my university program. No upload button on any browser or device.

No VPN. No AdBlock. I do keep getting pulled up for captcha on my IP for some reason across the web, despite no robots and completely malware-free.

None of my webcams can focus on my student ID well enough to be read. Website won't use camera on iOS, despite having permissions. Literally have no way of verification if there's no manual method here. What a stupid system.

Need urgent assistance for this issue.

**Reply by YerkinBS** [reply]
fr, such a big organization having stupid system like this, and can't fix this shit problem for like a 1+ year.

**Comment by sharklayer** [comment, marked as answer]
They just don't care, i already requested 26 times and got reject them all... i tried to send every document it was possible to me to take a picture and tried to take a picture in various ways, tried even send a print screen of my academic status directly from my college website. It has almost a month that i started requesting and i can't believe that i'm still begging for this shit.

**Reply by wahab890** [reply]
same, I tried taking a picture using my laptop cam and it was really annoying, I had to use face my phone screen to the pc and vice versa and it never worked, they must fix this issue.

**Reply by SapiensWilson** [reply]
Delightful, they sound fun.

**Comment by navjeet101709** [comment]
Same problem please tell me how to solve this

**Reply by Kvothe1979** [reply]
Doesn´t work

**Reply by sharklayer** [reply]
I don't think anyone here solved the problem, they just gave up... like me.

**Comment by snowy-shadow** [comment]
I took a screenshot of my enrollment letter and that did the trick

**Reply by mutigaj1** [reply]
you're so lucky. I have 8 tries on EVERYTHING,

**Comment by shamnad-sherief** [comment]
Take a picture as shown in the page. Then after an upload option can be seen on the bottom right corner. You may need to add your face with the ID proof . So posing with ID Card helps!!

**Comment by Harly-1506** [comment]
Don't use your laptop's camera, use a high-resolution phone camera to capture the most clear information. And if your student ID is not working, contact the school to provide a student identification with the date of issue and the school's signature

**Comment by epicseven-cup** [comment]
I like how it always register to my front camera on the application website

**Reply by uniyall** [reply]
yeah and when I select back camera, it doesnt work (iphone 13)

**Comment by hmtcelik** [comment]
still cant see upload button, any advice ?

**Reply by YerkinBS** [reply]
same. i'm so irritated, shit corporation

**Comment by ardantus** [comment]
You just need to drag and drop the PDF or JPG file onto the camera image, there's no need to take a photo with the camera, but the file that has been prepared will be automatically uploaded.

**Reply by uniyall** [reply]
it prompts -> File Uploads are not allowed

**Comment by Imadnajam** [comment]
If there's no longer an upload button visible, there might be an alternative method or a different location for uploading files.

**Comment by ahmetqunay** [comment]


**Reply by Olayiwolaaa** [reply]
If you're having trouble uploading your transcript to GitHub's Student Developer Pack, there could be a few reasons for this:

- **File Format and Size:** Ensure that your transcript file meets the required format and size specifications. GitHub may have restrictions on the file format (e.g., PDF, JPEG) and size limit for uploads.

- **Network or Server Issues:** Sometimes, issues with the network connection or GitHub's servers can prevent file uploads. Try uploading the transcript again at a later time or from a different network connection to see if the issue persists.

- **Permissions:** Make sure you have the necessary permissions to upload files to the GitHub Student Developer Pack application. If you're experiencing issues, check your account settings and permissions to ensure you have the required access.

- **Browser or Device Compatibility:** Ensure that you're using a supported browser and device to upload the transcript. Some browsers or devices may not be fully compatible with GitHub's file upload functionality.

- **Temporary Glitch:** Occasionally, temporary glitches or bugs in the GitHub platform can affect file uploads. If you encounter an error, try refreshing the page or clearing your browser's cache and cookies before attempting to upload the transcript again.

**Reply by JoaVitoTavares** [reply]
Capture a high-quality screenshot of the PDF document that effectively serves its purpose for me.

**Comment by ghost** [comment]
I'm having issues with this aswell, I'm a remote student on a comp-sci programme (at an established University in Sweden)  but we don't have any "physical" enrollment evidence, everything is digital. From my course certificates to my mecenat/student card. Our certificates have information on them on how to verify them aswell with the national student registry... I've tried taking a pic of my screen with the webcam but it's not remotely legible, especially if I have to fit the entire doc on screen. Why can't we just upload the protected pdf?

**Comment by FlorianN4** [comment]
You can use OBS virtual camera to take a picture, it works for me now ! ;)

**Comment by fri3s3n** [comment]
It seems to let you upload an image if you are present at the university campus 
My suggestions:
1. If you study in person. Go to campus and use your university's network to access the application page and try again. 
2. If you are an online student, it might help to send your details to someone present at your schools indicated campus and have them log-in and apply for you. you can change your login details right after. 
Otherwise Get a virtual camera like OBS, DroidCam etc and use that alongside your browser to enable you take photos with your back camera. 

I hope one of the above works, if not then I'm sorry you just have to wait till there is a better solution by the GitHub education team.

**Comment by mirzaabdullaht** [comment]
![Uploading image.png…]()


Hey everyone,
I wanted to share my experience with getting the GitHub Student Developer Pack. I ran into the same problem many of you are facing - there was no option to upload documents, and when I tried using the live camera, my application kept getting rejected.

Here's what I did:

I tried applying from different devices
I emailed GitHub's customer support
I even used a MacBook because its camera is way better than other laptops
My student ID was a bit tricky because it had two pages - one with my name and the other with the expiry date. After a bunch of rejections and following their suggestions, I gave it another shot with the MacBook. It took 5 rejections over a few days, but on my 6th try, it finally worked!

Now I'm just waiting for the benefits to kick in, which should take about 3 days.

Some tips from my experience:

Don't give up. Keep trying until you get approved.
Make sure to apply before you graduate. I only had 2 months left on my student ID.
Try to get the clearest picture possible of your ID.
I know how frustrating it is when you're trying to upload a clear document but can't, and you keep getting rejected. But hang in there!

My GitHub education request finally got approved.

Good luck to all of you trying to get your GitHub Education Pack :)

**Reply by AshkanArabim** [reply]
At least your ID has an expiration date. All I can provide is a screenshot of my enrollment status or course schedule, but that gets auto-rejected by their dumb AI.

Did GitHub's customer support get back to you though? If they did, I could get somewhere with that.

**Reply by mirzaabdullaht** [reply]
Yeah, GitHub support get back to me but that was the auto response..
Bro you can DM me on insta/LinkedIn.

So we can discuss the solution of it maybe.

**Comment by roa9618** [comment]
If you're a MacBook and iPhone user and it's hard to upload images for the above reasons, I'm sharing my own tips.

It has the ability to use the iPhone as a webcam for your MacBook.

After accessing the web on the MacBook, you can take photos using the iPhone by selecting the iPhone from the camera selection.

I took a picture and uploaded the certificate of enrollment using the iPhone and it was approved.
